### DISCUSSION TITLE
Some more colors

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1169] [Team Posts]
As came up just now in a bug discussion, I was having the wrong colors for the new Flat theme.

I was indeed missing the highlight for current track so I was looking around struggling to find how to get it back.
Kybos, as you mentioned I should load the colors from the /usr/local something DarkFlat.cfg.
I don't see how I can do that, I need to copy it by hand?
If that is the case, can we add a 'Reset colors to theme default' button maybe? 

Secondly, is it possible to do something about the colors of the buttons at the top of file dialogs? 
![bild](https://user-images.githubusercontent.com/4476983/84590733-5f180500-ae39-11ea-8d79-0e518ec9e129.png)
You can't really see what they do. A kde issue maybe..

**Comment by kybos** [comment]
There is a basic problem with the colours in that they are stored in the central user config file. They should be stored in a theme specific file in the user themes folder to avoid unwanted effects. I would like to change that but am a little afraid of backwards compatibility issues.

I already implemented several workarounds for this problem so in practice there should be no real issues with that for normal users. The problem that you have (and I had too when working on the PR) comes from changing between master and my PR while keeping the Dark Flat theme - master was missing all the new colour definitions so defaults were used. 

Fix 1:
Appearance->Colours->Load all..., then select "All files" below in the "Files of type" combo box, and select Dark Flat.cfg in the Global area (I guess we should put the .cfg extension in the default...).

Or Fix 2:
Switch to another theme (Appearance->Styles->Color schemes), then delete the file Dark Flat.cfc from your home config/themes folder, and switch back to Dark Flat theme.

As for the dark icons in the File selector popup: I am not sure we can influence that, it seems to be kind of a system dialog, but I will have a look. It's not a KDE issue - it looks a little better on my system, but not by much. But it's not directly related to my new theme, it's the same with the existing Dark Theme.

**Comment by spamatica** [comment]
I had already tried fix 2 but must have messed something up as I didn't get it to work.
Fix 1 seems to work fine though and is pretty much the solution I asked for, just wasn't aware of it. 
Thanks!

**Comment by terminator356** [comment]
On fix 1: I gave the custom colour files their own extension: *.cfc

**Comment by kybos** [comment]
> On fix 1: I gave the custom colour files their own extension: *.cfc

I agree it's a good idea to have an own extension for the colours (I used it too for the theme-specific user color config files), but then also the global theme colour files should be changed from .cfg to .cfc for consistent handling and to avoid unwanted side effects like the above (.cfg colour files being hidden from the user in the file dialog).

**Comment by kybos** [comment]
> Secondly, is it possible to do something about the colors of the buttons at the top of file dialogs?

It took some research in the Qt sources to make it possible, but now it's in master. Should look like this:

![image](https://user-images.githubusercontent.com/10009541/84778608-84487700-afe3-11ea-9d3f-ea4723d90538.png)

(These are standard Qt icons)
